test/integration/subreddit_edit_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class SubredditEditTest < ActionDispatch::IntegrationTest
+
+  def setup
+    @user = users(:josh)
+    @subreddit = subreddits(:sports)
+  end
+
+  test "should not edit subreddit" do
+    get edit_subreddit_path(@subreddit)
+    assert_template 'subreddits/edit'
+    patch subreddit_path(@subreddit), subreddit: {
+                                      name: "   ",
+                                      title: "   ",
+                                      description: "yup" }
+    assert_template 'subreddits/edit'
```

Comment:
What if the view logic was revised one day to still redirect to the edit template even after a successful template? This test would still pass, even if an edit was accidentally allowed. Can you think of a more direct and robust way to test if an edit went through?